galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/BaseTestRunner.java

```diff
@@ -166,12 +164,19 @@ private void loadOverrideProperties(Properties overrideProperties,
                                         IDynamicStatusStoreService dss) throws TestRunException {
         //*** Load the overrides if present
         try {
-            String prefix = "run." + run.getName() + ".override.";
-            Map<String, String> runOverrides = dss.getPrefix(prefix);
-            for(Entry<String, String> entry : runOverrides.entrySet()) {
-                String key = entry.getKey().substring(prefix.length());
-                String value = entry.getValue();
-                overrideProperties.put(key, value);
+            // The overrides DSS property contains a map of overrides, separated by newline characters in the form:
+            // dss.framework.run.<run-name>.overrides=override1=value1\noverride2=value2
```

Comment:
Should the value be base64-encoded?

Reply:
Values could contain newline characters. So using a \n as a separator isn't going to work well I think.

Should we use json ?

Reply:
Changed to a JSON array, so the property will now be in the following form:

```properties
dss.framework.run.C123.overrides=[{"key": "my.first.override", "value": "abc"}, {"key": "my.second.override", "value": "123"}]
```